ffs/types.go

```diff
@@ -226,6 +226,9 @@ type HotConfig struct {
 
 // Validate validates a HotConfig.
 func (hc HotConfig) Validate() error {
+	if !hc.Enabled {
+		return nil
+	}
```

Comment:
Cleaning up and making the validators a little more smart. If something isn't enabled, then you can easily say it is valid. Same below.